cmd/network/status.go

```diff
@@ -3,40 +3,158 @@ package network
 import (
 	"context"
 	"fmt"
+	"strings"
+	"time"
 
 	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
 
 	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
 
 	"github.com/oasisprotocol/cli/cmd/common"
 	cliConfig "github.com/oasisprotocol/cli/config"
 )
 
-var statusCmd = &cobra.Command{
-	Use:   "status",
-	Short: "Show the current status of the node and the network",
-	Args:  cobra.NoArgs,
-	Run: func(cmd *cobra.Command, args []string) {
-		cfg := cliConfig.Global()
-		npa := common.GetNPASelection(cfg)
+func getParatimeName(cfg *cliConfig.Config, id string) string {
+	for _, net := range cfg.Networks.All {
+		for ptName, pt := range net.ParaTimes.All {
+			if id == pt.ID {
+				return (ptName)
+			}
+		}
+	}
+	return ("unknown")
+}
+
+const (
+	formatJSON = "json"
+)
 
-		// Establish connection with the target network.
-		ctx := context.Background()
-		conn, err := connection.Connect(ctx, npa.Network)
-		cobra.CheckErr(err)
+var (
+	outputFormat string
 
-		ctrlConn := conn.Control()
+	statusCmd = &cobra.Command{
+		Use:   "status",
+		Short: "Show the current status of the node and the network",
+		Args:  cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := cliConfig.Global()
+			npa := common.GetNPASelection(cfg)
 
-		nodeStatus, err := ctrlConn.GetStatus(ctx)
-		cobra.CheckErr(err)
+			// Establish connection with the target network.
+			ctx := context.Background()
+			conn, err := connection.Connect(ctx, npa.Network)
+			cobra.CheckErr(err)
 
-		nodeStr, err := common.PrettyJSONMarshal(nodeStatus)
-		cobra.CheckErr(err)
+			ctrlConn := conn.Control()
 
-		fmt.Println(string(nodeStr))
-	},
-}
+			nodeStatus, err := ctrlConn.GetStatus(ctx)
+			cobra.CheckErr(err)
+
+			switch outputFormat {
+			case formatJSON:
+				nodeStr, err := common.PrettyJSONMarshal(nodeStatus)
+				cobra.CheckErr(err)
+
+				fmt.Println(string(nodeStr))
+			default:
+				fmt.Println("=== NETWORK STATUS ===")
+				fmt.Printf("Network:                  %s", npa.PrettyPrintNetwork())
+				fmt.Println()
+
+				fmt.Printf("Node's ID:                %s", nodeStatus.Identity.Node)
+				fmt.Println()
+
+				fmt.Printf("Core version:             %s", nodeStatus.SoftwareVersion)
+				fmt.Println()
+
+				fmt.Println()
+				fmt.Printf("=== Consensus ===")
+				fmt.Println()
+
+				fmt.Printf("  Status:                 %s", nodeStatus.Consensus.Status.String())
```

Comment:
I would remove all the leading indents in the Consensus and ParaTimes section, to save horizontal space.

```suggestion
				fmt.Printf("Status:                 %s", nodeStatus.Consensus.Status.String())
```